scripts/src/javatests/org/oppia/android/scripts/docs/KDocValidityCheckTest.kt

```diff
@@ -154,6 +272,22 @@ class KDocCheckTest {
     assertThat(outContent.toString().trim()).isEqualTo(KDOC_CHECK_PASSED_OUTPUT_INDICATOR)
   }
 
+  @Test
+  fun testKDoc_injectPrimaryConstructor_noSpecialBehaviour_checkShouldPass() {
```

Comment:
_withKdoc?

Ditto for other passing cases (to contrast with the 'withoutKdoc' tests). Similarly, I suggest separating out "_withoutKdoc" in those test names to be a bit clearer that it's a separate condition being implemented.

Reply:
Done.